src/Behat/Testwork/Output/Formatter.php

```diff
@@ -30,6 +30,13 @@
     public function getName();
 
     /**
+     * Returns file name.
+     * @param string $suite
+     * @return string
+     */
+    public function generateFilenameFor($suite);
```

Comment:
I'm not quite clear on the best way forward with this -- I see that the PassthroughFormatter is another Formatter that I should be concerned with. Should I call the generateFilenameFor() method only when I know the $formatter is the NodeEventListeningFromatter class?